### PR TITLE
[kube-prometheus-stack] Add labels to template for prometheus

### PR DESCRIPTION
**Pull request by Leweyy**
Signed-off-by: Lewis Brimfield <lewis.brimfield@wejo.com>

#### What this PR does / why we need it:
Adds option to add labels to prometheus

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.3.0
-appVersion: 0.43.2
+version: 12.4.1
+appVersion: 0.44.0
```
Comment (gkarthiks) [inline charts/kube-prometheus-stack/Chart.yaml:22]: Please do not change the appVersion. This corresponds to the mage version [here](https://github.com/prometheus-community/helm-charts/blob/64e66a78a379eed1b80fb1e8633838de6647d53f/charts/kube-prometheus-stack/values.yaml#L1416)

**Review by monotek — CHANGES_REQUESTED** [review]
Please  update version to 12.5.0

**Review by monotek — APPROVED** [review]

**Review by vsliouniaev — APPROVED** [review]

**Comment by gkarthiks** [comment]
@Leweyy Can you please rebase to resolve the conflicts?

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.10.6
```
Comment (gkarthiks) [inline charts/kube-prometheus-stack/Chart.yaml:21]: @Leweyy Can you please rebase to resolve the conflicts?

**Pull request closed without merging**